tests/framework/extensions/workloads/pods/scheduling.go

```diff
@@ -0,0 +1,240 @@
+package pods
```

Comment:
Yes I completely agree, after looking over the structs generated in v3 why don't we use those instead of copying over the structs from [api](https://github.com/kubernetes/api/tree/v0.27.1)/[core](https://github.com/kubernetes/api/tree/v0.27.1/core)/[v1](https://github.com/kubernetes/api/tree/v0.27.1/core/v1) ?
That way we are not unnecessarily copying.

Reply:
should be addressed now, thanks for the suggestion!